bitcoin/src/Bitcoin/Transaction/Taproot.hs

```diff
@@ -145,34 +121,34 @@ leafHash leafVersion leafScript =
 
 -- | Representation of a full taproot output.
 data TaprootOutput = TaprootOutput
-    { taprootInternalKey :: PubKey
+    { taprootInternalKey :: PubKeyXO
```

Comment:
Yeah, this seems right. The spec says that the internal key is the 32 byte x-only keys.

Reply:
When I originally wrote this code, I tried to keep `XOnlyPubKey` out of the public API since it was not clear when the upstream would actually end up supporting BIP340 operations.  This is an appropriate change.